share/spice/cryptocurrency/cryptocurrency.js

```diff
@@ -0,0 +1,292 @@
+        "mst": true,
+        "myr": true,
+        "nmc": true,
+        "nvc": true,
+        "nxt": true,
+        "pot": true,
+        "ppc": true,
+        "qrk": true,
+        "utc": true,
+        "wdc": true,
+        "xpm": true,
+        "xpy": true,
+        "xrp": true
+    }
+    
+    // Some naming exceptions. For example, "gbp" doesn't map to the "gb" asset.
+    // We need this hash so that we know that "gbp" will get the "uk" asset.
+    var currency2country_translate = {
+        'gb': 'uk'
+    };
+    
+    // Resize the size of the outer container if the content of the inner container
+    // overflows.
+    function resize() {        
+        var resultHeight = $(".zci--cryptocurrency-result").outerHeight();
+        
+        if(resultHeight > 65) {
+            $(".zci--cryptocurrency-container").css("height", "9em");
+        } else {
+            $(".zci--cryptocurrency-container").css("height", "5em");
+        }
+    }
+    
+    // Change the look of the mobile view if the content overflows.
+    function resizeMobile() {
+        var tileHeight = $(".zci--cryptocurrency .tile--s").outerHeight();
+
+        if(tileHeight > 155) {
+            $(".zci--cryptocurrency .tile--s").addClass("large").removeClass("small");
+        } else {
+            $(".zci--cryptocurrency .tile--s").addClass("small").removeClass("large");
+        }
+    }
+    
+    function decimalPlaces(number) {
+        return (number.toString()).replace(/^-?\d*\.?|0+$/g, '').length;
+    }
+    
+    env.ddg_spice_cryptocurrency = function(api_result){
+
+        if (!api_result) {
+            return Spice.failed('cryptocurrency');
+        };
+        
+        var results = [],
+            ticker = api_result.ticker,
+            rows = api_result.rows,
+            convertedAmount = 0,
+            rate = "",
+            inverseRate = "",
+            timestamp = "",
+            timestr = "",
+            cryptoDate = "",
+            cryptoTime = "",
+            script = $('[src*="/js/spice/cryptocurrency/"]')[0],
+            source = $(script).attr("src");
+        
+        if (typeof ticker !== 'undefined') {
+            
+            // Get amount from original query
+            var query = source.match(/\/ticker\/(?:.*)\/(.+)/)[1],
+                queryAmount = parseFloat(decodeURIComponent(query)),
+                // Calculate price, rates, and amounts
+                base = api_result.ticker.base,
+                target = api_result.ticker.target,
+                price = parseFloat(api_result.ticker.price),
+                results = api_result,
+                convertedAmount = queryAmount * price,
+                rate = "1 " + target + " = " + formatNumber((1 / price), base, 8) + " " + base,
+                inverseRate = "1 " + base + " = " + formatNumber(price, target, 8) + " " + target,
+                // Format Time and Date
+                timestamp = (new Date(api_result.timestamp*1000)).toISOString(),
```

Comment:
Just noticed that this is breaking IE8. It looks like `toISOString` is only supported in IE9+. I'll try and fix with Moment.js.